lib/chai/utils/proxify.js

```diff
@@ -64,7 +64,7 @@ module.exports = function proxify (obj, nonChainableMethodName) {
         }
       }
 
-      return target[property];
+      return Reflect.get(target, property);
```

Comment:
I think that by using `Reflect` we get more semantic code, for me it just feels like a good practice and a more functional way to invoke the language's internal operations.

However, since it won't have any impact and it's just a matter of personal preference I'd be happy with either options.